src/components/ProfilePhoto.js

```diff
@@ -0,0 +1,5 @@
+import image from "../profile-pictures/Rose.jpeg";
+
+export function ProfilePhoto() {
+  return <img src={image} className="profile-picture"></img>;
```

Comment:
- `img` elements are self-closing.
- `alt` attribute is a compulsory HTML attribute. It must always be present.